code/modules/organs/limbs.dm

```diff
@@ -460,7 +460,7 @@ Note that amputating the affected organ does in fact remove the infection from t
 		if (antibiotics < MIN_ANTIBIOTICS)
 			//Infected wounds raise the organ's germ level.
 			if (W.germ_level)
-				germ_level += min(1,round(W.germ_level/35))
+				germ_level += max(1,round(W.germ_level/35))
```

Comment:
a max is not consistent with the next code.

If you have no antibiotics (with current code), you can get from 1 to infinity germ level. With antibiotics, it's 1.

With your code, having no antibiotics is better than having some, since it's between 0 and 1 without antibiotic and 1 with

Reply:
Nah, you got it backwards. Old code: no antibiotics then 0 to 1, yes antibiotics then probably 1. New code: no anti then 1 to infinity, yes anti then probably 1.  min(1, x) isn't gonna give you more than one.

Reply:
ah you're right